server/js/sockets.js

```diff
@@ -115,6 +117,8 @@ sockets.init = function socketInit(server) {
     */
     mqttClient.subscribe('/v3/camera/recording/status/primary');
     mqttClient.subscribe('/v3/camera/recording/status/secondary');
+    mqttClient.subscribe('/v3/camera/video-feed/status/primary');
```

Comment:
I probably should have thought of this in the last PR, but can we subscribe using the single-topic wildcard? i.e. `/v3/camera/video-feed/status/+`

Reply:
Done
1eb90fb